src/lib/utils.ts

```diff
@@ -47,19 +49,13 @@ export function formatSeedId(id: string): string {
   return id.substring(0, 6) + "…" + id.substring(id.length - 6, id.length);
 }
 
-export function formatRadicleId(id: string): string {
-  assert(isRadicleId(id));
+export function formatRepositoryId(id: string): string {
```

Comment:
I think this function could be a bit more similar to `formatNodeId`, it isn't very clear from the function why we do `substring(0, 10)` and if we pass a RID without a `rad:` prefix this functions doesn't behave as needed.
But this can be done in a separate PR

Reply:
fb32bc381be89b503d67bfd6e7edada9fca6c02c

Reply:
I think this looks very nice.
So a minor nit, and I know you don't like indirection.. but wdyt about having a `formatRadicleId` and a `formatNodeId` wrapper around `formatId`?

E.g.
```
export function formatRepositoryId(id: string): string {
  return formatId(id, "repository");
}
```

So we don't have to specify on every format what we want to format but it's already in the function name?

Reply:
126ecb52184b44df5ec45ff7ea7a5eb38f2d737a